app/src/main/java/com/winlator/xconnector/XOutputStream.java

```diff
@@ -9,7 +9,7 @@
 
 public class XOutputStream {
     private static final byte[] ZERO = new byte[64];
-    private ByteBuffer buffer;
+    public ByteBuffer buffer;
```

Comment:
You've made this public, but aren't accessing it anywhere in this PR.

Is this on purpose?